scripts/qa_crowbarsetup.sh

```diff
@@ -3922,6 +3922,61 @@ EOF
     safely zypper -n in ca-certificates-suse
 }
 
+function ceph_testsuite_configure_storage3
+{
+    # this is the configuration for the tests from the storage3 branch
+    yaml_allnodes=`echo $cephmons $cephosds | sed "s/ /\n/g" | sed "s/\..*//g" | sort -ru`
+    yaml_osds=`echo $cephosds | sed "s/ /\n/g" | sed "s/\..*//g" | sort -ru`
```

Comment:
this does not work in case there are multiple osds per node. see the code in line 3974 how to do it properly